server/src/test/scala/org/scassandra/server/cors/CorsSupportTest.scala

```diff
@@ -0,0 +1,58 @@
+package org.scassandra.server.cors
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest._
+import org.scassandra.server.ScassandraConfig
+import org.scassandra.server.priming.cors.CorsSupport
+import spray.http._
+import spray.routing.HttpService
+import spray.testkit.ScalatestRouteTest
+
+
+class CorsSupportTest extends FlatSpec
+  with ScalatestRouteTest
+  with HttpService
+  with LazyLogging
+  with CorsSupport
+  with Matchers {
+  def actorRefFactory = system
+
+  val testRoute = path("test") {
```

Comment:
Yes, that was what I thought too. For what I know, I would have to change the `val's` to `var's` on ScassandraConfig... but, that would violate the immutable property of the Config! So we would have to add some kind of Mock around it, which seems overkill..